replication/ulyanov_et_al_2016/main.py

```diff
@@ -0,0 +1,235 @@
+import os
+from argparse import Namespace
+from os import path
+
+import torch
+
+from pystiche.image import write_image
+from pystiche.optim import OptimLogger
+from pystiche_papers.ulyanov_et_al_2016 import (
+    ulyanov_et_al_2016_dataset,
+    ulyanov_et_al_2016_image_loader,
+    ulyanov_et_al_2016_images,
+    ulyanov_et_al_2016_stylization,
+    ulyanov_et_al_2016_texture_synthesis,
+    ulyanov_et_al_2016_training,
+)
+from pystiche_papers.utils import save_state_dict
+
+
+def training_texture(args):
+    textures = (
+        "cezanne",
+        "bricks",
+        "pebble",
+        "pixels",
+        # "peppers",
+    )
+
+    dataset = ulyanov_et_al_2016_dataset(
+        path.join(args.dataset_dir, "texture"),
+        impl_params=args.impl_params,
+        instance_norm=args.instance_norm,
+    )  # FIXME: right dataset -> random from Imagenet (current examples from caltech256)
```

Comment:
Isn't that irrelevant from the implementation standpoint if we are using an `ImageFolderDataset`? Nevertheless, we should document this later on.